src/sessions/screen-share-session-handler.ts

```diff
@@ -14,7 +14,7 @@ export default class ScreenShareSessionHandler extends BaseSessionHandler {
   }
 
   // TODO: someday we should do media right before the session accept once we get away from media presence
```

Comment:
I know this `TODO` isn't from your commit, but isn't this taken care of in the new version of the streaming-client?

Reply:
I believe we can change this now, yes. I'll open a ticket for that.